Makefile

```
@@ -361,13 +361,23 @@ manifests.charts:
 
 KONG_OPERATOR_CHART_DIR = $(PROJECT_DIR)/charts/kong-operator
 
+.PHONY: ensure.go.pkg.downloaded.kubernetes-configuration
+ensure.go.pkg.downloaded.kubernetes-configuration:
+	@go mod download $(KUBERNETES_CONFIGURATION_PACKAGE)@$(KUBERNETES_CONFIGURATION_VERSION)
+
+.PHONY: ensure.go.pkg.downloaded.gateway-api
+ensure.go.pkg.downloaded.gateway-api:
+	@go mod download $(GATEWAY_API_PACKAGE)@$(GATEWAY_API_VERSION)
+
 .PHONY: manifests.charts.kong-operator.crds.operator
 manifests.charts.kong-operator.crds.operator: kustomize
+	@$(MAKE) ensure.go.pkg.downloaded.kubernetes-configuration
```

```suggestion
manifests.charts.kong-operator.crds.operator: kustomize ensure.go.pkg.downloaded.kubernetes-configuration
```
We can represent it directly with dependencies, without needing to use `@$(MAKE)`. WDTY?

---

Good point, 2709fde956b9efe6b19ef7f787319a4c3f3e4330 does that. PTAL.